pkg/governance-scripts/contracts/xxxx-gauge-adder-migration-v3-to-v4/GaugeAdderMigrationCoordinator.sol

```diff
@@ -30,6 +30,7 @@ contract GaugeAdderMigrationCoordinator is BaseCoordinator {
     ILiquidityGaugeFactory public immutable polygonRootGaugeFactory;
     ILiquidityGaugeFactory public immutable arbitrumRootGaugeFactory;
     ILiquidityGaugeFactory public immutable optimismRootGaugeFactory;
+    ILiquidityGaugeFactory public immutable gnosisRootGaugeFactory;
```

Comment:
We also have a zkEVM Root Gauge now (or will very soon). Should we add that one,?

Reply:
good question.

I'm moving forward with this PR to work on the ones on top, but I'll have this in mind to correct it before release if necessary.

This can be added by governance later on in any case.